share/nitweb/views/doc.html

```diff
@@ -0,0 +1,39 @@
+<div class='container-fluid'>
+
+	<div ng-if='error' class='alert alert-danger' role='alert'>
+		<span class='glyphicon glyphicon-exclamation-sign' aria-hidden='true'></span>
+		<span class='sr-only'>Error:</span>
+		<span ng-switch='error.status'>
+			<span ng-switch-when='404'>Entity <code>{{entityId}}</code> not found!</span>
```

Comment:
Professional error messages do not end by an exclamation point.

* https://material.google.com/style/writing.html#writing-capitalization-punctuation
* https://material.google.com/patterns/errors.html#errors-app-errors
* https://msdn.microsoft.com/en-us/library/windows/desktop/ms679325(v=vs.85).aspx
* https://msdn.microsoft.com/en-us/library/dn742471.aspx
* http://www.openoffice.org/specs/collaterals/guides/text-style-guide.html#Exclamation
* http://stackoverflow.com/questions/3354867/does-anyone-use-a-style-guide-for-error-messages